cmd/panacead/cmd/decrypt.go

```diff
@@ -64,7 +81,7 @@ func Command(defaultNodeHome string) *cobra.Command {
 	return cmd
 }
 
-func Decrypt(clientCtx client.Context, cmd *cobra.Command, filePath, keyName string, dealID uint64) ([]byte, error) {
+func decrypt(clientCtx client.Context, keyName string, encryptedData []byte, oraclePubKey string, nonce []byte) ([]byte, error) {
```

Comment:
And I made this `decrypt` function smaller, which just decrypt the cipher text using ingredients.